src/components/image-preview/__snapshots__/image-preview.test.tsx.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Animal should image src tbe this injected as a props 1`] = `ShallowWrapper {}`;
```

Comment:
boilerplate ?

Reply:
Yes. Should not be versioned. 
Indeed, Animals are out of Cheerz domain